tools/ci_build/github/azure-pipelines/linux-gpu-ci-pipeline.yml

```diff
@@ -57,3 +71,52 @@ jobs:
   - task: mspremier.PostBuildCleanup.PostBuildCleanup-task.PostBuildCleanup@3
     displayName: 'Clean Agent Directories'
     condition: always()
+
+- job: Linux_Test
+  timeoutInMinutes: 60
+  workspace:
+    clean: all
+  pool: centos7gpu
+  dependsOn:
+  - Linux_Build
+  steps:
+  - task: DownloadPipelineArtifact@2
+    displayName: 'Download Pipeline Artifact'
+    inputs:
+      buildType: 'current'
+      artifactName: 'drop-linux'
+      targetPath: '$(Build.BinariesDirectory)/Release'
+
+  - task: CmdLine@2
+    inputs:
+      script: |
+         set -e -x
+         #We assume the machine doesn't gcc and python development header files
+         sudo rm -f /build /onnxruntime_src
+         sudo ln -s $(Build.SourcesDirectory) /onnxruntime_src
+         python3 -m pip uninstall -y ort-gpu-nightly ort_nightly onnxruntime onnxruntime-gpu -qq
+         cp $(Build.SourcesDirectory)/tools/ci_build/github/linux/docker/scripts/manylinux/requirements.txt $(Build.BinariesDirectory)/requirements.txt
+         #Test ORT with the latest ONNX release.
+         sed -i 's/git+http:\/\/github\.com\/onnx\/onnx.*/onnx==1.9.0/' $(Build.BinariesDirectory)/requirements.txt
+         python3 -m pip install -r $(Build.BinariesDirectory)/requirements.txt
+         python3 -m pip install $(Build.BinariesDirectory)/Release/dist/*.whl
+         ln -s /data/models $(Build.BinariesDirectory)
+         cd $(Build.BinariesDirectory)/Release
+         #Restore file permissions
+         xargs -a $(Build.BinariesDirectory)/Release/perms.txt chmod a+x
+         cd $(Build.SourcesDirectory)/java
+         /usr/local/gradle/bin/gradle "cmakeCheck" "-DcmakeBuildDir=$(Build.BinariesDirectory)/Release" "-DUSE_CUDA=1"
+         cd /tmp
+         python3 $(Build.SourcesDirectory)/tools/ci_build/build.py \
+              --build_dir $(Build.BinariesDirectory) --cmake_generator Ninja \
+              --config Release --test \
+              --skip_submodule_sync \
+              --build_shared_lib \
+              --parallel \
+              --build_wheel \
+              --enable_onnx_tests --use_cuda --cuda_version=11.1 --cuda_home=/usr/local/cuda-11.1 --cudnn_home=/usr/local/cuda-11.1 \
+              --enable_pybind --build_java --ctest_path ''
```

Comment:
>--ctest_path '' [](http://example.com/codeflow?start=43&length=15)

maybe another option like --no_ctest would be clearer?
at least documenting that an empty string is valid and what that means would help

Reply:
Added document there.

In the future I plan to completely remove the dependency on ctest.  I can't see much value of it. And it disallows us splitting our pipelines to build/test two parts. Because ctest hardcodes all the paths names to its configure files. Once I moved the build dir, it stops working.  But I need to upload the built binaries from build machine to another test machine.